base/util_testing.go

```diff
@@ -2,6 +2,8 @@ package base
 
 import (
 	"bytes"
+	random "crypto/rand"
```

Comment:
Would suggest aliasing this to `cryptorand` instead, to avoid confusion with the already imported `math/rand`.

```suggestion
	cryptorand "crypto/rand"
```

Reply:
Renamed random alias to cryptorand to avoid confusion with the already imported math/rand.